packages/eslint-plugin/docs/rules/no-useless-template-literals.md

```diff
@@ -0,0 +1,53 @@
+---
+description: 'Disallow unnecessary template literals.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/no-useless-template-literals** for documentation.
+
+This rule reports template literals that can be simplified to a normal string literal.
+
+## Examples
+
+<!--tabs-->
+
+### ❌ Incorrect
+
+```ts
+const ab1 = `${'a'}${'b'}`;
+const ab2 = `a${'b'}`;
+
+const stringWithNumber = `1 + 1 = ${2}`;
+
+const stringWithBoolean = `${'true is '}${true}`;
+
+const string = 'a';
+const wrappedString = `${string}`;
+
+declare const intersectionWithString: string & { _brand: 'test-brand' };
+const wrappedIntersection = `${intersectionWithString}`;
+```
+
+### ✅ Correct
+
+```ts
+const ab1 = 'ab';
+const ab2 = 'ab';
+
+const stringWithNumber = '1 + 1 = 2';
+
+const stringWithBoolean = 'true is true';
+
+const string = 'a';
```

Comment:
[Docs] It's sometimes confusing for folks to see the name of a primitive used as a variable name. 

```suggestion
const text = 'a';
```

Reply:
Done